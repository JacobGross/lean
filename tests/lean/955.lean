-*- mode: compilation; default-directory: "~/projects/lean/tests/lean/run/" -*-
Compilation started at Thu Feb  4 14:53:07

/home/leo/projects/lean/bin/lean -Dpp.width=120  /home/leo/projects/lean/tests/lean/run/955.lean
/home/leo/projects/lean/tests/lean/run/955.lean:31:32: error: type mismatch at definition 'append_auxH', has type
  @vectorH.{?M_1}
    (nat.addl nat.zero
       (nat.addl nat.zero
          (nat.addl nat.zero
             (nat.addl nat.zero
                (nat.addl nat.zero
                   (nat.addl nat.zero
                      (nat.addl nat.zero
                         (nat.addl nat.zero
                            (nat.addl nat.zero
                               (nat.addl nat.zero
                                  (nat.addl nat.zero
                                     (nat.addl nat.zero
                                        (nat.addl nat.zero
                                           (nat.addl nat.zero
                                              (nat.addl nat.zero
                                                 (nat.addl nat.zero
                                                    (nat.addl nat.zero
                                                       (nat.addl nat.zero
                                                          (nat.addl nat.zero
                                                             (nat.addl nat.zero
                                                                (nat.addl nat.zero
                                                                   (nat.addl nat.zero
                                                                      (nat.addl nat.zero
                                                                         (nat.addl nat.zero
                                                                            (nat.addl nat.zero
                                                                               (nat.addl nat.zero
                                                                                  (nat.addl nat.zero
                                                                                     (nat.addl nat.zero
                                                                                        (nat.addl nat.zero
                                                                                           (nat.addl nat.zero
                                                                                              (nat.addl nat.zero
                                                                                                 (nat.addl nat.zero
                                                                                                    (nat.addl nat.zero
                                                                                                       (nat.addl
                                                                                                          nat.zero
                                                                                                          (nat.addl
                                                                                                             nat.zero
                                                                                                             (nat.addl
                                                                                                                nat.zero
                                                                                                                (nat.addl
                                                                                                                   nat.zero
                                                                                                                   (nat.addl
                                                                                                                      nat.zero
                                                                                                                      (nat.addl
                                                                                                                         nat.zero
                                                                                                                         (nat.addl
                                                                                                                            nat.zero
                                                                                                                            (nat.addl
                                                                                                                               nat.zero
                                                                                                                               (nat.addl
                                                                                                                                  nat.zero
                                                                                                                                  (nat.addl
                                                                                                                                     nat.zero
                                                                                                                                     (nat.addl
                                                                                                                                        nat.zero
                                                                                                                                        (nat.addl
                                                                                                                                           nat.zero
                                                                                                                                           (nat.addl
                                                                                                                                              nat.zero
                                                                                                                                              (nat.addl
                                                                                                                                                 nat.zero
                                                                                                                                                 (nat.addl
                                                                                                                                                    nat.zero
                                                                                                                                                    (nat.addl
                                                                                                                                                       nat.zero
                                                                                                                                                       (nat.addl
                                                                                                                                                          nat.zero
                                                                                                                                                          (nat.addl
                                                                                                                                                             nat.zero
                                                                                                                                                             (nat.addl
                                                                                                                                                                nat.zero
                                                                                                                                                                (nat.addl
                                                                                                                                                                   nat.zero
                                                                                                                                                                   (nat.addl
                                                                                                                                                                      nat.zero
                                                                                                                                                                      (nat.addl
                                                                                                                                                                         nat.zero
                                                                                                                                                                         (nat.addl
                                                                                                                                                                            nat.zero
                                                                                                                                                                            (nat.addl
                                                                                                                                                                               nat.zero
                                                                                                                                                                               (nat.addl
                                                                                                                                                                                  nat.zero
                                                                                                                                                                                  (nat.addl
                                                                                                                                                                                     nat.zero
                                                                                                                                                                                     (nat.addl
                                                                                                                                                                                        nat.zero
                                                                                                                                                                                        (nat.addl
                                                                                                                                                                                           nat.zero
                                                                                                                                                                                           (…
                                                                                                                                                                                              …))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))
    ?M_4
but is expected to have type
  @vectorH.{?M_1}
    (nat.addl nat.zero
       (nat.addl nat.zero
          (nat.addl nat.zero
             (nat.addl nat.zero
                (nat.addl nat.zero
                   (nat.addl nat.zero
                      (nat.addl nat.zero
                         (nat.addl nat.zero
                            (nat.addl nat.zero
                               (nat.addl nat.zero
                                  (nat.addl nat.zero
                                     (nat.addl nat.zero
                                        (nat.addl nat.zero
                                           (nat.addl nat.zero
                                              (nat.addl nat.zero
                                                 (nat.addl nat.zero
                                                    (nat.addl nat.zero
                                                       (nat.addl nat.zero
                                                          (nat.addl nat.zero
                                                             (nat.addl nat.zero
                                                                (nat.addl nat.zero
                                                                   (nat.addl nat.zero
                                                                      (nat.addl nat.zero
                                                                         (nat.addl nat.zero
                                                                            (nat.addl nat.zero
                                                                               (nat.addl nat.zero
                                                                                  (nat.addl nat.zero
                                                                                     (nat.addl nat.zero
                                                                                        (nat.addl nat.zero
                                                                                           (nat.addl nat.zero
                                                                                              (nat.addl nat.zero
                                                                                                 (nat.addl nat.zero
                                                                                                    (nat.addl nat.zero
                                                                                                       (nat.addl
                                                                                                          nat.zero
                                                                                                          (nat.addl
                                                                                                             nat.zero
                                                                                                             (nat.addl
                                                                                                                nat.zero
                                                                                                                (nat.addl
                                                                                                                   nat.zero
                                                                                                                   (nat.addl
                                                                                                                      nat.zero
                                                                                                                      (nat.addl
                                                                                                                         nat.zero
                                                                                                                         (nat.addl
                                                                                                                            nat.zero
                                                                                                                            (nat.addl
                                                                                                                               nat.zero
                                                                                                                               (nat.addl
                                                                                                                                  nat.zero
                                                                                                                                  (nat.addl
                                                                                                                                     nat.zero
                                                                                                                                     (nat.addl
                                                                                                                                        nat.zero
                                                                                                                                        (nat.addl
                                                                                                                                           nat.zero
                                                                                                                                           (nat.addl
                                                                                                                                              nat.zero
                                                                                                                                              (nat.addl
                                                                                                                                                 nat.zero
                                                                                                                                                 (nat.addl
                                                                                                                                                    nat.zero
                                                                                                                                                    (nat.addl
                                                                                                                                                       nat.zero
                                                                                                                                                       (nat.addl
                                                                                                                                                          nat.zero
                                                                                                                                                          (nat.addl
                                                                                                                                                             nat.zero
                                                                                                                                                             (nat.addl
                                                                                                                                                                nat.zero
                                                                                                                                                                (nat.addl
                                                                                                                                                                   nat.zero
                                                                                                                                                                   (nat.addl
                                                                                                                                                                      nat.zero
                                                                                                                                                                      (nat.addl
                                                                                                                                                                         nat.zero
                                                                                                                                                                         (nat.addl
                                                                                                                                                                            nat.zero
                                                                                                                                                                            (nat.addl
                                                                                                                                                                               nat.zero
                                                                                                                                                                               (nat.addl
                                                                                                                                                                                  nat.zero
                                                                                                                                                                                  (nat.addl
                                                                                                                                                                                     nat.zero
                                                                                                                                                                                     (nat.addl
                                                                                                                                                                                        nat.zero
                                                                                                                                                                                        (nat.addl
                                                                                                                                                                                           nat.zero
                                                                                                                                                                                           (…
                                                                                                                                                                                              …))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))
    (@vector.append_aux.{?M_1+1} Type.{?M_1} nat.zero
       (nat.addl nat.zero
          (nat.addl nat.zero
             (nat.addl nat.zero
                (nat.addl nat.zero
                   (nat.addl nat.zero
                      (nat.addl nat.zero
                         (nat.addl nat.zero
                            (nat.addl nat.zero
                               (nat.addl nat.zero
                                  (nat.addl nat.zero
                                     (nat.addl nat.zero
                                        (nat.addl nat.zero
                                           (nat.addl nat.zero
                                              (nat.addl nat.zero
                                                 (nat.addl nat.zero
                                                    (nat.addl nat.zero
                                                       (nat.addl nat.zero
                                                          (nat.addl nat.zero
                                                             (nat.addl nat.zero
                                                                (nat.addl nat.zero
                                                                   (nat.addl nat.zero
                                                                      (nat.addl nat.zero
                                                                         (nat.addl nat.zero
                                                                            (nat.addl nat.zero
                                                                               (nat.addl nat.zero
                                                                                  (nat.addl nat.zero
                                                                                     (nat.addl nat.zero
                                                                                        (nat.addl nat.zero
                                                                                           (nat.addl nat.zero
                                                                                              (nat.addl nat.zero
                                                                                                 (nat.addl nat.zero
                                                                                                    (nat.addl nat.zero
                                                                                                       (nat.addl
                                                                                                          nat.zero
                                                                                                          (nat.addl
                                                                                                             nat.zero
                                                                                                             (nat.addl
                                                                                                                nat.zero
                                                                                                                (nat.addl
                                                                                                                   nat.zero
                                                                                                                   (nat.addl
                                                                                                                      nat.zero
                                                                                                                      (nat.addl
                                                                                                                         nat.zero
                                                                                                                         (nat.addl
                                                                                                                            nat.zero
                                                                                                                            (nat.addl
                                                                                                                               nat.zero
                                                                                                                               (nat.addl
                                                                                                                                  nat.zero
                                                                                                                                  (nat.addl
                                                                                                                                     nat.zero
                                                                                                                                     (nat.addl
                                                                                                                                        nat.zero
                                                                                                                                        (nat.addl
                                                                                                                                           nat.zero
                                                                                                                                           (nat.addl
                                                                                                                                              nat.zero
                                                                                                                                              (nat.addl
                                                                                                                                                 nat.zero
                                                                                                                                                 (nat.addl
                                                                                                                                                    nat.zero
                                                                                                                                                    (nat.addl
                                                                                                                                                       nat.zero
                                                                                                                                                       (nat.addl
                                                                                                                                                          nat.zero
                                                                                                                                                          (nat.addl
                                                                                                                                                             nat.zero
                                                                                                                                                             (nat.addl
                                                                                                                                                                nat.zero
                                                                                                                                                                (nat.addl
                                                                                                                                                                   nat.zero
                                                                                                                                                                   (nat.addl
                                                                                                                                                                      nat.zero
                                                                                                                                                                      (nat.addl
                                                                                                                                                                         nat.zero
                                                                                                                                                                         (nat.addl
                                                                                                                                                                            nat.zero
                                                                                                                                                                            (nat.addl
                                                                                                                                                                               nat.zero
                                                                                                                                                                               (nat.addl
                                                                                                                                                                                  nat.zero
                                                                                                                                                                                  (nat.addl
                                                                                                                                                                                     nat.zero
                                                                                                                                                                                     (nat.addl
                                                                                                                                                                                        nat.zero
                                                                                                                                                                                        (…
                                                                                                                                                                                           …))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))
       (@vector.nil.{?M_1+1} Type.{?M_1})
       ?M_3)
The following identifier(s) are introduced as free variables by the left-hand-side of the equation:
  v'

Compilation exited abnormally with code 1 at Thu Feb  4 14:53:09
