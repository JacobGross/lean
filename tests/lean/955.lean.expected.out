955.lean:1:1: error: unexpected token
955.lean: error: 955.lean:1:21: error: unexpected token
